frontend/src/layout/FeaturePreviews/FeaturePreviews.tsx

```diff
@@ -82,7 +71,14 @@ function FeaturePreview({ feature }: { feature: EnrichedEarlyAccessFeature }): J
     return (
         <div>
             <div className="flex items-center justify-between">
-                <h4 className="font-semibold mb-0">{name}</h4>
+                <div className="flex items-center gap-2">
```

Comment:
```suggestion
                <div className="flex items-center gap-1">
```

The buttons feel a little bit far away. Not a strong opinion, though:

| Before | After |
|--------|--------|
| ![CleanShot 2025-02-11 at 05 07 53@2x](https://github.com/user-attachments/assets/533d7f04-6c5d-4236-91a2-57e28494003c) | ![CleanShot 2025-02-11 at 05 08 21@2x](https://github.com/user-attachments/assets/523fa1f4-cfb7-49e3-8b12-e1c23b52bc78) |